scripts/preprocessing/04b-run_ssp.py

```diff
@@ -21,6 +23,30 @@
 
 
 @failsafe_run(on_error=on_error)
+
+def _get_global_reject_ssp(raw):
```

Comment:
nice! we should have this for ICA too! (but don't worry, I can add it myself later ;))